STUDIO/org.openl.rules.project/src/org/openl/rules/project/xml/OpenLSerializationException.java

```diff
@@ -0,0 +1,13 @@
+package org.openl.rules.project.xml;
+
+import org.openl.exception.OpenlNotCheckedException;
+
+public class OpenLSerializationException extends OpenlNotCheckedException {
```

Comment:
What is the reason to create one more exception? Can be it replaced by standard Java exceptions, like IllegalArgumentException or something else?